From b57dde69dc80b151844e0762a2ae6bca3ba86d95 Mon Sep 17 00:00:00 2001
From: Kefu Chai <tchaikov@gmail.com>
Date: Mon, 15 Jul 2024 11:10:09 +0800
Subject: [PATCH] fix build with fmt v11

fmt::join() was moved into fmt/ranges.h since fmt 11, so let's
include this header for using it.

Signed-off-by: Kefu Chai <tchaikov@gmail.com>
---
 src/common/common.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/common/common.h b/src/common/common.h
index 68a129fab2..b5e0c8a899 100644
--- a/src/common/common.h
+++ b/src/common/common.h
@@ -35,6 +35,7 @@
 
 #include <fmt/format.h>
 #include <fmt/ostream.h>
+#include <fmt/ranges.h>
 
 #include <boost/algorithm/string.hpp>
 #include <boost/multiprecision/cpp_int.hpp>
-- 
GitLab

